### PR TITLE
add some version info

### PR DESCRIPTION
**Pull request by raybellwaves**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [NA] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This is motivated as a package which uses shap isn't building. The reason for this is unclear but it's bringing in an old version of slicer (https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=254696&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=921 and  https://github.com/conda-forge/explainerdashboard-feedstock/pull/13#issuecomment-749069126)

It may be beneficial for this recipe to pin some requirements so the recipe matches that of the package requirement versions e.g. https://github.com/slundberg/shap/blob/master/setup.py#L220

Here's i've pined `tqdm > 4.25.0` and `slicer == 0.0.7`.

### CODE REVIEW

**Comment by raybellwaves** [comment]
@conda-forge-admin, please rerender

**Comment by raybellwaves** [comment]
Friendly ping @slundberg, @mrandrewandrade for a review here

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,9 +32,9 @@ requirements:
     - scikit-image
     - scipy
     - matplotlib-base
-    - tqdm
+    - tqdm >4.25.0
     - ipython
-    - slicer
+    - slicer ==0.0.7
```
Comment (xhochy) [inline recipe/meta.yaml:37]:
```suggestion
    - slicer 0.0.7
```

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

**Pull request merged**